src/ApiCheck.Task/ApiCheckTask.cs

```diff
@@ -1,24 +1,19 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
 using System.IO;
 using System.Reflection;
-using System.Runtime.InteropServices;
 using Microsoft.Build.Framework;
-using Microsoft.Build.Utilities;
-using Microsoft.Extensions.CommandLineUtils;
 
 namespace Microsoft.AspNetCore.BuildTools.ApiCheck.Task
 {
     /// <summary>
     /// An MSBuild task that acts as a shim to <c>Microsoft.AspNetCore.BuildTools.ApiCheck.exe compare ...</c> or
     /// <c>dotnet Microsoft.AspNetCore.BuildTools.ApiCheck.dll compare ...</c>.
     /// </summary>
-    public class ApiCheckTask : ToolTask
+    public class ApiCheckTask : ApiCheckTasksBase
```

Comment:
Whatever is in `ApiCheckTasksBase` after addressing comments in `ApiCheckGenerateTask` can be removed here too.